### PR TITLE
Stop screenshot plugin from overwriting pictures

### LINKED ISSUE #4374: Screenshots taken within the same second overwrite previous screenshots in that second
If I click the screenshot hotkey or button multiple times within a second, the last screenshot taken within that second will be the only one present in the screenshots folder because it overwrites any previous screenshots of the same name already present.

Note: This is true in any other scenario where an existing screenshot has the same name as a new one. (DMM level-up screenshots, for instance)

### PR DESCRIPTION
**Pull request by Noremac201**
fixes #4374

Simply added more precise seconds to the file name.

### CODE REVIEW

**Comment by Nightfirecat** [comment]
This fixes the linked issue, but not the more general "Screenshots of same name overwrite each other" issue. Personally, I'd like to see a fix that works like most filesystems handle duplicate file names--appending a counter to the screenshot, like  `Screenshot Name (1).png`

**Comment by Noremac201** [comment]
Right you are, 
I've updated the PR to fix the more general issue.

File: runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java [hunk 1]
```diff
@@ -107,7 +107,7 @@
 	private static final HttpUrl IMGUR_IMAGE_UPLOAD_URL = HttpUrl.parse("https://api.imgur.com/3/image");
 	private static final MediaType JSON = MediaType.parse("application/json");
 
-	private static final DateFormat TIME_FORMAT = new SimpleDateFormat("yyyy-MM-dd_HH-mm-ss");
+	private static final DateFormat TIME_FORMAT = new SimpleDateFormat("yyyy-MM-dd_HH-mm-ss.SSS");
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java]: With the change to loop over filenames, this probably isn't needed.

**Comment by Noremac201** [comment]
Removed the unnecessary code, and updated the commit.

Thanks for taking a look.

**Pull request merged**